### ISSUE TITLE
Use CSS property font-display 

### ISSUE THREAD

**Issue by dgt41** [joomla/40-backend-template#336]
### Steps to reproduce the issue

Granular control over font loading.
Support is not bad
<img width="1286" alt="screen shot 2018-02-16 at 10 43 14" src="https://user-images.githubusercontent.com/3889375/36301790-5f3367ee-1306-11e8-93b4-99397a5b917e.png">

### Required changes

I guess use `supports` in combination with `font-display`.

Also the whole font strategy **should** be based on the best practices as described here: https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/webfont-optimization#webfonts_and_the_critical_rendering_path

**Comment by brianteeman** [comment]
Yes please. I came to this tracker to add a comment about it and was glad to see it here

https://developers.google.com/web/updates/2016/02/font-display
